crates/matrix-sdk/src/sliding_sync.rs

```diff
@@ -1406,6 +1410,11 @@ impl SlidingSyncView {
             changed = true;
         }
 
+        if self.rooms_count.get() != Some(rooms_count) {
+            self.rooms_count.replace(Some(rooms_count));
+            changed = true
```

Comment:
```suggestion
            changed = true;
```

Reply:
why isn't this something rustfmt picks up? same for missing `;` on return ...

Reply:
I opened a feature request for it a while ago, maybe the new rustfmt team will implement it next year 🤷🏼 https://github.com/rust-lang/rustfmt/issues/5363